app/models/collection.rb

```diff
@@ -86,6 +92,29 @@ def submitters
     User.with_role(:submitter, self)
   end
 
+  # Permit a User to receive notification messages for members of this Collection
+  # @param user [User]
+  def enable_messages_for(user:)
+    raise(ArgumentError, "User #{user.uid} is not an administrator for this collection #{title}") unless user.super_admin? || user.can_admin?(self)
```

Comment:
I believe this will also work, because [super admins have administrative](https://github.com/pulibrary/pdc_describe/blob/a46d7752190be72e777d0616e97b836240aae7e3/app/models/user.rb#L72) access to any collection.
```suggestion
    raise(ArgumentError, "User #{user.uid} is not an administrator for this collection #{title}") unless user.can_admin?(self)
```